core/app/modules/kit-library/assets/js/pages/index/index.js

```diff
@@ -210,12 +211,22 @@ export default function Index( props ) {
 				</Grid>
 				<Content className="e-kit-library__index-layout-main">
 					<>
-						{ isLoading && __( 'Loading...', 'elementor' ) }
-						{ isError && __( 'An error occurred', 'elementor' ) }
+						{ isLoading && <PageLoader /> }
+						{
+							isError && <ErrorScreen
+								title={ __( 'Something went wrong.', 'elementor' ) }
+								description={ __( 'Nothing to worry about, use 🔄 on the top right to try again. If the problem continues, head over to the Help Center.', 'elementor' ) }
```

Comment:
Hmm, do we use emojis here?
I'm not sure it's a good idea for UI consistency, since each OS has a different design:
https://emojipedia.org/counterclockwise-arrows-button/

Reply:
It is ok for me.

Reply:
@matipojo what do you think?

Reply:
Let try. I 💗 emojis.